shared/node.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/pokt-network/pocket/state_machine"
 	"github.com/pokt-network/pocket/telemetry"
 	"github.com/pokt-network/pocket/utility"
+	"go.uber.org/multierr"
```

Comment:
cc @bryanchriswhite 

@gokutheengineer is actually adding it right now. Let's add a TODO to remove this one we move to Go 1.2: https://stackoverflow.com/a/74645028/768439

Reply:
🙌